### DISCUSSION TITLE
Select Component Scrolls to the Top automatically when a list is long

### DISCUSSION THREAD

**Discussion by sunnysetia93** [quasarframework/quasar#12712] [General - Components / Directives / etc]
**Problem Statement:** Select Component Scrolls to the Top automatically when a list is long

**Quasar Version:** 

"@quasar/app": "^2.0.0"
"quasar": "^1.0.0",
"@quasar/extras": "^1.0.0",

**Attaching a video for reference.**

https://user-images.githubusercontent.com/6140084/157378505-6e688b85-bc75-4491-8464-b6f993f42ea3.mov

**Comment by metalsadman** [comment]
please provide a reproduction.
